### ISSUE TITLE
Feature: Migration geoservice IGN

### ISSUE THREAD

**Issue by flob38** [c2corg/c2c_ui#3746]
**Community discussion about this feature (mandatory)**
https://forum.camptocamp.org/t/bascule-des-geoservices/324653

**Is your feature request related to a problem? Please describe.**
It will be a bug if nothing change before 15/01/24 : no more IGN layers on map

**Describe the solution you'd like**
Change url of map's layers on UI
[src/components/map/map-layers.js](https://github.com/c2corg/c2c_ui/blob/5686ae334c4ced7c054080dda7682cf0cf0ae76f/src/components/map/map-layers.js#L45)

**Comment by florentcadot** [comment]
Hi @flob38 !
I cannot access https://forum.camptocamp.org/t/bascule-des-geoservices/324653 (I suppose the page is private?)

![Screenshot from 2023-12-23 11-33-10](https://github.com/c2corg/c2c_ui/assets/52289507/009ff9d2-d2ac-4897-b6c5-edd263ca2854)

**Comment by flob38** [comment]
Should be ok now, I added you to the dev group on the forum :)

**Comment by florentcadot** [comment]
Seems to have been mostly done here: https://github.com/c2corg/c2c_ui/commit/143e68dc4a17b2908e9a2e4b2781c9989d8b1ba4

Just wondering about : https://github.com/c2corg/c2c_ui/blob/3e3f07f1c55ccc9ae804b8d01eed7ef1615fb189/src/components/yeti/map-layers/WinterRouteLayer.vue#L38 

I guess it should be ` url: 'https://data.geopf.fr/private/wms-v/ows?apikey=blabla'`  (with *blabla* the keyname according to https://geoservices.ign.fr/documentation/services/utilisation-sig/tutoriel-qgis/gpf-wms-wmts-donneesnonlibres)
cf doc here https://geoservices.ign.fr/bascule-vers-la-geoplateforme "Correspondances des URL par géoservice".

**Comment by eddy-geek** [comment]
@florentcadot  I agree we should try to migrate TRACERANDOHIVERNALE, but I think it's not available yet. 
when when I compare the "capabilities" of the old (1), new private (2) and new public (3) urls:

(1) https://wxs.ign.fr/an7nvfzojv5wa96dsga5nk8w/geoportail/v/wms?SERVICE=WMS&VERSION=1.3.0&REQUEST=GetCapabilities
(2) https://data.geopf.fr/private/wms-v/ows?apikey=ign_scan_ws&service=wms&version=1.3.0&request=GetCapabilities
(3) https://data.geopf.fr/wms-v/ows?service=wms&version=1.3.0&request=GetCapabilities

only the first one mentions this layer:

```xml
14201:        <Name>TRACERANDOHIVERNALE</Name>
14202:        <Title>TRACERANDOHIVERNALE</Title>
14203-        <Abstract>Layer-Group type layer: TRACERANDOHIVERNALE</Abstract>
```

If I take a layer that **is declared** in (2), it works. example:

`curl 'https://data.geopf.fr/private/wms-v/ows?apikey=ign_scan_ws&REQUEST=GetMap&SERVICE=WMS&VERSION=1.3.0&FORMAT=image%2Fpng&STYLES=&TRANSPARENT=true&LAYERS=ADMINEXPRESS_2023-12-19&WIDTH=256&HEIGHT=256&CRS=EPSG%3A3857&BBOX=682429.7885300554%2C5576845.58368646%2C684875.7734351811%2C5579291.568591585' \
  -H 'user-agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36' \
  --output ign-ADMINEXPRESS_2023-12-19-sample-tile.png`

If I just replace `ADMINEXPRESS_2023-12-19` with `TRACERANDOHIVERNALE` I get 
` <h1>Error 403</h1>  <p>Unauthorized access to the offering TRACERANDOHIVERNALE</p>`

which is actually a generic error for any random name!

→ so I think it's just not migrated yet. (hard to be sure as geoportail itself still uses the old URLs...)

**Comment by eddy-geek** [comment]
I asked IGN and they confirmed "not yet"

> La publication de cette couche est bien prévue, vous pouvez consulter depuis la page [Bascule](https://geoservices.ign.fr/bascule-vers-la-geoplateforme), la [Liste des ressources de services Géoplateforme](https://geoservices.ign.fr/sites/default/files/2023-12/Tableau%20suivi%20services%20web%2022-12-2023.csv)
 
> La couche TRACERANDOHIVERNALE est indiquée comme non diffusée pour le moment. Elle le sera avant l'arrêt de l'infrastructure du Géoportail.

**Comment by florentcadot** [comment]
Okay ! Thank you very much for all those details @eddy-geek, it's very clear :smile: 
So it makes sense to keep this issue opened until TRACERANDOHIVERNALE is published!
